src/apis/LogBox.md

```diff
@@ -0,0 +1,53 @@
+---
+id: apis/LogBox
+title: LogBox
+officialDoc: https://reactnative.dev/docs/debugging#logbox
+---
+## [LogBox](https://reactnative.dev/docs/debugging#logbox)
```

Comment:
This title is redundant, page will generate it from `title`.
Also you can then update the `###` to `##`

Reply:
Removed it.

Reply:
I still see this. And I don't see the other title adjusted neither.

Reply:
Must not have pushed the edit. I see it here now.